### DISCUSSION TITLE
Where is javadoc api documentation for current (i.e. v12) primefaces version

### DISCUSSION THREAD

**Discussion by pki-acs** [primefaces/community#1484] [PrimeFaces] [answered]
I wanted to look up the reference documentation for  org.primefaces.model.ScheduleModel mentioned here: https://primefaces.github.io/primefaces/13_0_0/#/components/schedule

I'm not able to find it there, and google search only brings javadoc documentation for primefaces up to 8.0. Where can I find structured and complete API reference for current primefaces version?

**Comment by NicolaIsotta** [comment, marked as answer]
You can find javadocs for all published versions on javadoc.io: https://javadoc.io/doc/org.primefaces/primefaces/12.0.0/index.html

It think would be nice to have direct links from to the docs the javadocs, not sure if there's a way to do that automagically...

**Reply by pki-acs** [reply]
thank you. Exactly what I was looking for.
